server/src/main/java/com/linecorp/centraldogma/server/CentralDogmaConfig.java

```diff
@@ -86,6 +87,8 @@ public final class CentralDogmaConfig {
 
     private static final Logger logger = LoggerFactory.getLogger(CentralDogmaConfig.class);
 
+    private static final Pattern PREFIX_PATTERN = Pattern.compile("^[a-z0-9_]+$");
```

Comment:
Optional) Maybe just me, but I find it common to also include dashes in property names

```suggestion
    private static final Pattern PREFIX_PATTERN = Pattern.compile("^[a-z0-9_-]+$");
```

Reply:
The private key starts with `-`, so I intentionally didn't put it. But maybe it was too strict. Let me put it back. 😆